### ISSUE TITLE
[Bug]: One-click installation fails at first step

### ISSUE THREAD

**Issue by very-common** [stereum-dev/ethereum-node#1473]
### Describe the bug!

One-click installation for Staking fails at the step "Delete Node" and drops me back to the main screen (Select your Server Configuration). I tried with all networks (Mainnet, Testnet), same result.

I could not find any logs (neither locally, nor on the server) - any suggestions?

Description:
- Stereum version: 2.0.0-rc.22
- Server is Ubuntu 22.04 (upgraded from fresh 20.04 installation)
- OS check was successful

### Expected Behaviour

I expected it to

### Help us reproduce your bug!

Server Setup:
1) Install Ubuntu 20.04
2) Upgrade to Ubuntu 22.04
3) Set up passwordless sudo for user

Stereum:
1) Log into server
2) 1 Click installation
3) Select any network (does not matter)
4) Select Staking
5) Next
6) Next
7) Next
8) Installation starts, step "Delete Node"
9) Screen flashes, I am back at the "Select your Server Configuration" screen

### Launcher Version

2.0.0-rc.22

### Desktop OS

Windows

### Node Server OS

Ubuntu 22.04

### Acknowledgment

- [X] I looked through the existing bugs of this project

**Comment by daGscheid** [comment]
Thanks for the report ❤️ We will look into it asap. We aware of the problem #1344  , but couldn't sucessfully reproduce it so far, so we might come back with some question 🙏

**Comment by NeoPlays** [comment]
Hey could you prived some logs? you can find them here:

   - on Linux: `~/.config/stereum-launcher/logs/main.log`
   - on macOS: `~/Library/Logs/stereum-launcher/main.log`
   - on Windows: `%USERPROFILE%\AppData\Roaming\stereum-launcher\logs\main.log`

**Comment by very-common** [comment]
Hi!
I am running Stereum on Windows.

There are not logs, not on my Desktop, not on the Ubuntu server. :(

**Comment by very-common** [comment]
Alternatively, I could pass the login info privately, so you can try to connect from a different device?

**Comment by daGscheid** [comment]
> Alternatively, I could pass the login info privately, so you can try to connect from a different device?

Not a huge fan of that idea for various security reasons *haha* and giving the nature of the error (Launcher <> Server connection), it probably won't help. Should you run into this again & can pass any info, please let us know 🙏

**Comment by very-common** [comment]
Also not a fan, but there are no logs whatsoever...

I mean, the problem is still unresolved, still can't set up the server.

**Comment by NeoPlays** [comment]
What you could try is running following command on your machine:
- open up a terminal and connect to your server via ssh
- run following command:`apt update && apt install -y software-properties-common &&  add-apt-repository --yes --update ppa:ansible/ansible && apt install -y pip ansible tar gzip wget git`
- this command installs necessary packages to run stereum, but sometimes this command requires user input and our automated script then fails.

I hope this helps.

**Comment by very-common** [comment]
I resolved it. 

tl;dr: Passwordless sudo configuration was wrong but Stereum did not catch it.

Here is a rundown:

* Ran your commands manually - did not resolve the issue 
* Upgraded to the next RC version of Stereum - did not resolve the issue
* With the upgraded version I did finally find logs on Windows. `stderr got data sudo: a terminal is required to read the password;`
* I re-checked /etc/sudoers to make sure passwordless sudo is on
* YES: Passwordless sudo was configured for my user but here is the problem: Group settings override individual user settings if they appear further down in the config file. Stereum OS check apparently does not catch that.

Suggestion
* Update Stereum OS check to run an actual passwordless sudo command
* Update the Stereum guide to explain how to configure passwordless sudo correctly

**Comment by daverolo** [comment]
Thank you @very-common for the detailed rundown.
> YES: Passwordless sudo was configured for my user but here is the problem: Group settings override individual user settings if they appear further down in the config file. Stereum OS check apparently does not catch that.

- We usually show a warning but do not block the user from installing. Do you mean there was no warning at all?
- Could you maybe provide an example /etc/sudoers file that we can use to reproduce the issue?

**Comment by very-common** [comment]
In /etc/sudoers the passwordless user should not appear *before* the group (it is part of), because the group setting overrides the user setting. Stereum only shows a warning if the NOPASSWD option is missing _altogether_.

**This will NOT work but Stereum shows NO warning:**
`# User privilege specification, notice how the user stereum appears BEFORE the sudo group
root	ALL=(ALL:ALL) ALL
stereum ALL=(ALL) NOPASSWD: ALL

# Allow members of group sudo to execute any command (this overrides the NOPASSWD option above)
%sudo	ALL=(ALL:ALL) ALL`

**This WILL work:**
`# User privilege specification
root	ALL=(ALL:ALL) ALL

# Allow members of group sudo to execute any command, user stereum appears AFTER the sudo group
%sudo	ALL=(ALL:ALL) ALL
stereum ALL=(ALL) NOPASSWD: ALL
`

**Comment by daverolo** [comment]
Great catch, thank you @very-common
Reproduced the issue and fixed it with #1487
